### PR TITLE
Default to Integrate out of plane angle on ALFView

### LINKED ISSUE #34626: ALFView - Default to Integrate Out of Plane Angle when selecting whole tube
**Description**
When the `Select whole tube` button is clicked in the ALFView interface, it is required to right click on the mini-plot and select `Integrate`, and then right click again to select the axis to be `Out of plane angle`. This makes the workflow more laborious and would be good if we didn't need to do these two steps.

**Describe any solutions you are considering**
When the `Select whole tube` button is clicked in the ALFView interface, the `Integrate` option and `Out of Plane Angle` axis should be selected by default. This will save two right clicks on the mini-plot to select `Integrate` and then the `Axis`.

Agreed this with David.

### PR DESCRIPTION
**Pull request by robertapplin**
**Description of work.**
This PR ensures the 'Out of plane angle' is selected as the axis on the mini plot in the Pick tab of the ALFView interface when the `Select whole tube` button is selected. It also ensures the `Integrate` option is selected by default.

This is by far the primary workflow on ALFView so it makes sense to have this as a default.

This saves having to do two right clicks and three left clicks.

**To test:**
Rather than doing:

1. Load data [ALF82301](file:///home/mlc47243/Documents/other-documents/ALFView/ALF82301.zip)
2. Select Pick tab. 
3. Expand plotting section.
4. Click `Select whole tube`
5. Right click mini plot, then click `Integrate`
6. Right click mini plot, then select `Axes`->`Out of plane angle`

you should now be able to do 

1. Load data [ALF82301](file:///home/mlc47243/Documents/other-documents/ALFView/ALF82301.zip)
2. Select Pick tab. 
3. Expand plotting section.
4. Click `Select whole tube`

and the plot axes should be 'Out of plane angle'

Fixes #34626 

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by MialLewis — CHANGES_REQUESTED** [review]
Code works and looks good - a nice improvement with the `enums`. Just one question.

**Comment by robertapplin** [comment]
Thanks, should be ready for another look @MialLewis

File: qt/widgets/instrumentview/src/InstrumentWidgetPickTab.cpp [hunk 1]
```diff
@@ -586,16 +597,15 @@ void InstrumentWidgetPickTab::removeCurve(const QString &label) {
 
 /**
  * Set the x units for the integrated tube plot.
- * @param units :: The x units in terms of TubeXUnits.
+ * @param units :: The x units in terms of IWPickXUnits.
  */
-void InstrumentWidgetPickTab::setTubeXUnits(int units) {
-  if (units < 0 || units >= DetectorPlotController::NUMBER_OF_UNITS)
```
[the remark this answers is not in the document]
Comment (robertapplin) [reply qt/widgets/instrumentview/src/InstrumentWidgetPickTab.cpp:592]: Good spot, I think this is still needed

**Review by MialLewis — APPROVED** [review]
Thanks for making the change - good to go.

**Pull request merged**